src/main/webapp/createYourOwn.html

```diff
@@ -22,6 +22,51 @@
             </nav>
         </div>
       </header>
+      <h1>New Initiative</h1>
+      <hr style="height:4px;background-color:#F8944C;border-width:0px;color:#F8944C;width:300px;margin-bottom:30px">
+      <div id="new-initiative">
+          <form id="newInitiativeForm" action="my-blobstore-url" method="POST" enctype="multipart/form-data">
+            <p class="initiative-input">Title:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-title">
+            <br/><br/>
+            <p class="initiative-input">Insert Initiative Image:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="file" name="initiative-image">
+            <br/><br/>
+            <p class="initiative-input">Location:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-location">
+            <br/><br/>
+            <p class="initiative-input">Start Date:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-start-date" value="MM/DD/YYYY">
+            <br/><br/>
+            <p class="initiative-input">End Date:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-end-date" value="MM/DD/YYYY">
+            <br/><br/>
+            <p class="initiative-input">Context:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input id="big-text-space" type="text" name="initiative-context">
+            <br/><br/>
+            <p class="initiative-input">Cause:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input id="big-text-space" type="text" name="initiative-cause">
+            <br/><br/>
+            <p class="initiative-input">Target Audience:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input id="big-text-space" type="text" name="initiative-target-audience">
+            <br/><br/>
+            <p class="initiative-input">Instructions:
```

Comment:
In CSS, you could style all img tags under the same ID or class selector altogether. That might reduce the redundancy in the styling for all these symbols.

Reply:
Fixed!